lectures/markov_chains_I.md

```diff
@@ -220,7 +220,7 @@ Then we can address a range of questions, such as
 We'll cover some of these applications below.
 
 (mc_eg3)=
-#### Example 3
+#### Example 3: Political Transition Dynamics
```

Comment:
Can you please review the section titles re: capitalisation

https://manual.quantecon.org/styleguide/writing.html#titles-and-headings